devicetypes/smartthings/zwave-fan-controller.src/zwave-fan-controller.groovy

```diff
@@ -123,13 +124,11 @@ def fanEvents(physicalgraph.zwave.Command cmd) {
 
 		def fanLevel = 0
 
-		// The GE, Honeywell, and Leviton treat 33 as medium, so account for that
-		if (1 <= rawLevel && rawLevel <= 32) {
-			fanLevel = 1
-		} else if (33 <= rawLevel && rawLevel <= 66) {
-			fanLevel = 2
-		} else if (67 <= rawLevel && rawLevel <= 100) {
-			fanLevel = 3
+		// The GE, Honeywell, and Leviton 3-Speed Fan Controller treat 33 as medium, so account for that
```

Comment:
If you are breaking the logic out in to helper functions, please move this comment in to the 3-speed function. Just put it right above the first `if` as it originally was here.

Reply:
Please move this to right above line 228.